myapp/spec/requests/task_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :request do
+  describe 'GET /index' do
+    let(:tasks) { create_list(:task, 10) }
+
+    it 'renders a successful response' do
+      get tasks_url
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'GET /new' do
+    it 'renders a successful response' do
+      get new_task_url
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'GET /edit' do
+    let(:task) { create(:task) }
+
+    it 'renders a successful response' do
+      get edit_task_url(task)
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'POST /create' do
+    context 'with parameters' do
+      let(:params) do
+        { task: {
+          title: 'task',
+          priority: 'low',
+          status: 'working',
+          description: 'task desu',
+          expires_at: Time.now + 1.week
```

Comment:
`Time.now`は環境変数(ENV['TZ'])、もしくはシステムのタイムゾーンを元に現在日時を取得するメソッドなので`Time.current`か`Time.zone.now`を使うほうが望ましいですー！

一週間後であれば`1.week.since`で表現できますー

```suggestion
          expires_at: 1.week.since
```

https://docs.ruby-lang.org/ja/latest/method/Time/s/now.html
https://api.rubyonrails.org/classes/Time.html#method-c-current
https://api.rubyonrails.org/classes/DateAndTime/Calculations.html#method-i-weeks_since

Reply:
同様の箇所がいくつかありますが、書くとコメントが多くなるので省略しますね